.github/workflows/command-update-ui.yml

```
@@ -0,0 +1,55 @@
+name: Command Update UI
+
+on: 
+  workflow_dispatch: 
+    inputs:
+      pr:
+        description: Number of the Pull Request
+        required: true
+      rust-version:
+        description: Version of rust. Example 1.70
+        required: false
+
+jobs:
+  set-image:     
+    runs-on: ubuntu-latest
+    outputs:
+      IMAGE: ${{ steps.set_image.outputs.IMAGE }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - id: set_image
+        run: cat .github/env >> $GITHUB_OUTPUT
+  cmd-update-ui:
+    needs: [set-image]
+    runs-on: arc-runners-polkadot-sdk-benchmark
```

```suggestion
    runs-on: arc-runners-polkadot-sdk-beefy
```

In this job we need a powerful machine to compile the tests fast so it's better to use `arc-runners-polkadot-sdk-beefy`